source/dub/dub.d

```diff
@@ -188,7 +188,12 @@ class Dub {
 			ps ~= defaultPackageSuppliers();
 
 		m_packageSuppliers = ps;
-		m_packageManager = new PackageManager(m_dirs.localRepository, m_dirs.systemSettings);
+		m_packageManager = new PackageManager(m_dirs.localRepository, m_dirs.systemSettings, true);
```

Comment:
Could be a flag: `RefreshPackages.Yes`

Reply:
I'll actually remove that from the diff, as that was left over from an intermediate version. But, I agree that a concrete type is preferable here.